File: bandlog-bootstrap/src/main/java/net/effize/bandlog/api/config/JpaConfig.java
package net.effize.bandlog.api.config;

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EnableJpaAuditing
@EnableJpaRepositories(basePackages = "net.effize.bandlog")
@EntityScan(basePackages = "net.effize.bandlog")
public class JpaConfig {
}

File: src/main/kotlin/net/effize/bandlog/common/config/JpaConfig.kt
package net.effize.bandlog.common.config

import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@Configuration
@EnableJpaAuditing
@EnableJpaRepositories(basePackages = ["net.effize.bandlog"])
@EntityScan(basePackages = ["net.effize.bandlog"])
class JpaConfig
